### PR TITLE
Add run exports to libxmu

### PR DESCRIPTION
**Pull request by ehfd**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

Fixes run_import and increases build number. Requested by @xhochy

### CODE REVIEW

**Comment by ehfd** [comment]
@conda-forge-admin, please rerender

**Comment by ehfd** [comment]
@xhochy I can't rerender yet.

Edit: I can.

**Comment by ehfd** [comment]
@conda-forge-admin, please rerender

**Comment by ehfd** [comment]
@xhochy @pkgw Build should succeed.

**Review by xhochy — APPROVED** [review]
LGTM

**Comment by ehfd** [comment]
@pkgw Very thankful if this could be merged timely.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -2,23 +2,25 @@
...
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.bz2
-  url: https://www.x.org/releases/individual/{{ xorg_category }}/{{ xorg_name }}-{{ version }}.tar.bz2
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://www.x.org/releases/individual/{{ xorg_category }}/{{ xorg_name }}-{{ version }}.tar.gz
```
Comment (pkgw) [inline recipe/meta.yaml:14]: Why are we switching from the bz2 to the gzip here?

Comment (ehfd) [reply recipe/meta.yaml:14]: In the latest X.org releases

https://www.x.org/releases/individual/lib/

They've changed from bz2 & gz offerings to xz & gz offerings.

bz2 is no longer in recent releases and gz is the only format provided by X.org which is consistent.

Comment (ehfd) [reply recipe/meta.yaml:14]: @pkgw

**Comment by ehfd** [comment]
@conda-forge-admin, please rerender

**Comment by ehfd** [comment]
@conda-forge-admin, please rerender

**Comment by ehfd** [comment]
Everything works except that the ppc64le build doesn't start from the queue.

**Comment by xhochy** [comment]
My first approach would be to switch to cross-compiling here. Can you remove the provider line from the `conda-forge.yml` and extend the `build_platform` entry to:

```
build_platform: 
  osx_arm64: osx_64
  linux_ppc64le: linux_64
  linux_aarch64: linux_64
```

… and rerender?

**Comment by ehfd** [comment]
@conda-forge-admin, please rerender

**Comment by ehfd** [comment]
@xhochy For future reference, conda-forge.yml is indeed the file to add when I want new architectures (such as s390x) in my recipe?

**Comment by xhochy** [comment]
> @xhochy For future reference, conda-forge.yml is indeed the file to add when I want new architectures (such as s390x) in my recipe?

Yes!

File: recipe/meta.yaml (hunk 1 above)
Comment (ehfd) [reply recipe/meta.yaml:14]: We're ready when you're ready. @pkgw

**Comment by ehfd** [comment]
@pkgw Thank you!

**Pull request merged**